code/modules/unit_tests/cargo_crate_sanity.dm

```diff
@@ -0,0 +1,14 @@
+/**
+ * This unit test loops through all cargo crates that are available to purchase, and confirms that they're below the expected sanity minimum when sold.
+ * This prevents us from merging a crate that sells for more that it costs to buy.
+ */
+
+/datum/unit_test/cargo_crate_sanity
+
+/datum/unit_test/cargo_crate_sanity/Run()
+	for(var/datum/supply_pack/new_crate in subtypesof(/datum/supply_pack))
```

Comment:
This doesn't work, subtypesof gives you a list of, well, subtypes - not of objects. You need to init those yourself here, thus the name error and missing procs